packages/server/src/api/routes/tests/table.spec.ts

```diff
@@ -276,6 +276,35 @@ describe.each([
         })
     })
 
+    isInternal &&
+      it("shouldn't allow duplicate column names", async () => {
+        const saveTableRequest: SaveTableRequest = {
+          ...basicTable(),
+        }
+        saveTableRequest.schema["Type"] = {
+          type: FieldType.STRING,
+          name: "Type",
+        }
+        await config.api.table.save(saveTableRequest, {
+          status: 400,
+          body: {
+            message:
+              'Column "type" is duplicated - make sure there are no duplicate columns names, this is case insensitive.',
+          },
+        })
```

Comment:
I don't feel too strongly about this but I'd probably get rid of this section of the test. It doesn't test anything that the foo test below doesn't test and it depends on a detail of the basicTable method that may potentially change.

Reply:
Added some more usefulness to it - this checks for prohibited columns and then just straight duped columns, as well as confirming multiple can be errored on.